Formula/vowpal-wabbit.rb

```diff
@@ -26,14 +30,18 @@ def install
     mkdir "build" do
       system "cmake", "..", *std_cmake_args,
                             "-DBUILD_TESTS=OFF",
-                            "-DRAPIDJSON_SYS_DEP=ON"
+                            "-DRAPIDJSON_SYS_DEP=ON",
+                            "-DFMT_SYS_DEP=ON",
+                            "-DSPDLOG_SYS_DEP=ON",
+                            "-DBUILD_FLATBUFFERS=ON"
```

Comment:
It shouldn't though, it should use the brew dependency

Reply:
It does use the brew dependency (see line 18), this is an optional feature in the library that we are signalling should be used.

Reply:
Line 18 just adds the dependency, that doesn't mean it's used.
---
```
                            "-DSPDLOG_SYS_DEP=ON",
                            "-DBUILD_FLATBUFFERS=ON"
```
I was triggered by the difference between the settings here. But if you say this is how you enable the flatbuffers system dependency I'll trust you on that. The naming just doesn't make that clear.

Reply:
Yes, I agree the option name is unclear, sorry about that. I can confirm this is how the flatbuffer feature is turned on and that it does use the system dependency.